src/multi_vector_simulator/C0_data_processing.py

```diff
@@ -1011,6 +1016,7 @@ def define_source(
         MAXIMUM_CAP: {VALUE: None, UNIT: "?"},
         AGE_INSTALLED: {VALUE: 0, UNIT: UNIT_YEAR,},
         ENERGY_VECTOR: energy_vector,
+        EMISSION_FACTOR: emission_factor,
```

Comment:
Shouldn´t this also be a value-unit pair?
```
EMISSION_FACTOR: {VALUE: emission_factor, UNIT: ...},
```

Reply:
Yes, it is. The description in the docstring was wrong. Changed to
```
    emission_factor : dict
        Dict with a unit-value pair of the emission factor of the new asset
```